Documentation/ColumnsConfig/Type/Inline/Index.rst

```diff
@@ -11,39 +11,43 @@ IRRE / inline
     column type :ref:`file <columns-file>` instead.
 
 ..  versionadded:: 13.0
-    When using the `inline` type, TYPO3 takes care of
-    :ref:`generating the according database field <t3coreapi:auto-generated-db-structure>`.
-    A developer does not need to define this field in an extension's
+    When using the `inline` type, TYPO3
```

Comment:
Please adjust this for all types as once as a separate change, otherwise we have this wording and the old one in this manual together. Maybe as a follow-up in the next days, so this one gets in.

Reply:
Feel free to provide a PR ;-) noone is really going to read a reference from start to finish